docs/source/daml2js/index.rst

```diff
@@ -1,25 +1,25 @@
 .. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-``daml2ts`` - TypeScript code generator for DAML
+``daml2js`` - TypeScript code generator for DAML
```

Comment:
Somewhat similar question.  I think it’s slightly confusing to have this called `daml2js` while the command is called `daml codegen js`.